tests/OSPSuite.Starter/Tasks/DataRepositoryCreator.cs

```diff
@@ -70,36 +82,106 @@ private static IQuantity getQuantityFromOrganism(IContainer organism)
          return quantity1;
       }
 
-      public IEnumerable<DataRepository> CreateCalculationRepositories(int numberOfCalculations, IContainer model, int index)
+      public DataRepository CreateCalculationRepository(int numberOfCalculations, IContainer model, int index, int pointsPerCalculation)
       {
-         
          var repositoryName = $"Calculation Repository {index}";
-         var dataRepository = createRepository(numberOfCalculations, model, repositoryName, ColumnOrigins.Calculation);
+         var dataRepository = createRepository(numberOfCalculations, model, repositoryName, ColumnOrigins.Calculation, pointsPerCalculation);
 
-         yield return dataRepository;
+         return dataRepository;
       }
 
-      private DataRepository createRepository(int numberOfCalculations, IContainer model, string repositoryName, ColumnOrigins columnOrigins)
+      private DataRepository createRepository(int numberOfCalculations, IContainer model, string repositoryName, ColumnOrigins columnOrigins, int numberOfPointsPerCalculation)
       {
          var dataRepository = new DataRepository().WithName(repositoryName);
-         var baseGrid = createBaseGridWithManyPoints(dataRepository.Name);
+         var baseGrid = createBaseGridWithManyPoints(dataRepository.Name, numberOfPointsPerCalculation);
 
-         for (int i = 0; i < numberOfCalculations; i++)
+         for (var i = 0; i < numberOfCalculations; i++)
          {
-            dataRepository.Add(calculationColumnFor(baseGrid, i, model, columnOrigins));
+            dataRepository.Add(createPrimaryColumnFor(baseGrid, i, model, columnOrigins));
          }
          return dataRepository;
       }
 
-      public IEnumerable<DataRepository> CreateObservationRepositories(int numberOfObservations, IContainer model, int index)
+      private DataColumn createAuxiliaryColumn(DataColumn column, int index, IContainer model, AuxiliaryType columnType, ColumnOrigins columnOrigins, Func<double, float> calculateAuxiliaryFactor)
```

Comment:
```
GeoMean => value = Mean*Factor and Mean/Factor
ArithMEan => value = Mean + factor and Mean - factor
```

so you should only multiply with the factor for arithmean. Otherwise use the factor as is